extras/mmu/mmu.py

```diff
@@ -5006,12 +5012,16 @@ def _do_form_tip(self, test=False):
                 else:
                     self.log_trace(msg)
 
-            if not test and park_pos > self.toolhead_extruder_to_nozzle:
+            if reported and park_pos > self.toolhead_extruder_to_nozzle:
+                self.log_debug("park_pos (%.1fmm) is greater than 'toolhead_extruder_to_nozzle' distance of %.1fmm! Assuming cutter is located before extruder." % (park_pos, self.toolhead_extruder_to_nozzle))
+                before_extruder = True
+                filament_remaining = max(0., filament_remaining)
```

Comment:
I'm also not sure if that calculation for the remaining filament is correct. It worked for me, but I'm not sure if this will work if the cutter is located further away from the extruder entry.

Reply:
currently testing if removing the call to max works reliably, converted the PR to draft again because of that